velox/exec/tests/utils/OperatorTestBase.cpp

```diff
@@ -47,6 +47,27 @@ void OperatorTestBase::registerVectorSerde() {
 OperatorTestBase::~OperatorTestBase() {
   // Revert to default process-wide MappedMemory.
   memory::MappedMemory::setDefaultInstance(nullptr);
+  waitForAllTasksToBeDeleted();
+}
+
+void OperatorTestBase::waitForAllTasksToBeDeleted(uint64_t maxWaitUs) {
```

Comment:
Is there an assumption that tests never run in parallel?

Reply:
Yeah, I think gtest won't run more than one test cases in parallel in the same process? It is the caller's responsibility to ensure there is no more tasks to be created after this call point. We could use this in the middle of a test case execution as well. For example, the test has created some tasks and then wait for them all to be deleted before the rest of steps.